serverless/develop/serverless-kafka-developer.adoc

```diff
@@ -34,8 +38,25 @@ include::modules/specifying-sink-flag-kn.adoc[leveloffset=+3]
 // YAML
 include::modules/serverless-kafka-source-yaml.adoc[leveloffset=+2]
 
+[id="serverless-kafka-developer-broker"]
+== Using a Kafka broker
+
+:FeatureName: Kafka broker
+include::modules/technology-preview.adoc[leveloffset=+2]
+
+If a cluster administrator has configured your {ServerlessProductName} deployment to use Kafka broker as the default broker type, xref:../../serverless/develop/serverless-using-brokers.adoc#serverless-using-brokers-creating-brokers[creating a broker by using the default settings] creates a `KafkaBroker` object instead of a generic `Broker` object.
```

Comment:
There is no `KafkaBroker` type.

We use the `Broker` API.  The actual implementation of the API is choosen by annotation, as explained in this yaml example file:  https://github.com/openshift/openshift-docs/pull/40701/files#diff-2b26a1f5d6a577f6180d4aa69409357cbba931d400ed7f128122602bc106aed9R22